client/lib/js/react/components/bigchaindb_ledgerplugin.js

```diff
@@ -122,12 +122,28 @@ class BigchainDBLedgerPlugin extends EventEmitter2 {
     getConnectors() {
     }
 
+    /*
+    Initiates a ledger-local transfer.
+     */
     send(transfer) {
         return co.wrap(this._send).call(this, transfer);
     }
 
     * _send(transfer) {
-        
+        let res;
+        try {
+            res = yield request('assets_escrow', {
```

Comment:
I've created an issue for it.

https://github.com/bigchaindb/bigchaindb-examples/issues/19